### DISCUSSION TITLE
Table 11, Panel 1

### DISCUSSION THREAD

**Discussion by sandragesing** [cssi-pi-community/2022-meeting#28] [Panel 1 roundtable (Monday morning)]
Training of DEI and Code of Conducts is important to improve DEI in workforce development. I think it should be included in cybertraining grants - not only to increase DEI but the knowledge about it.

**Comment by DanielRPhillips** [comment]
## Notes from discussion at our table ##

We need to make sure that the university climate values DEI. This is a cultural, logistical, and career path issue. 

- CI DEI training that complements existing efforts at institutions would be useful to promote a positive culture within groups.
- Evaluation criteria that are flexible enough to represent different understandings of effective work are important.
- Flexibility in work arrangements to handle unexpected events are important at the institutional leve, e.g., parental leave.
- If institutions do not have policies that support a diverse workforce (e.g. parental leave, other benefits for post docs) NSF should perhaps refuse to allocate funding to those institutions.
- Specific solicitations for proposals with post docs as PIs (along with the lines of NIH K99-R00) with a transition to a CI professional position at the end would aid the pipeline.
- Such professional development pathways are particularly important to alert diverse members of our workforce to the possibilities in the field, when they may have particular reasons to feel discouraged and/or life situations that make them disinclined to “stick around” in academia
